syntax = "proto3";

import "mlagents_envs/communicator_objects/agent_info.proto";
import "mlagents_envs/communicator_objects/agent_action.proto";

option csharp_namespace = "MLAgents.CommunicatorObjects";
package communicator_objects;

message AgentInfoActionPairProto {
    AgentInfoProto agent_info = 1;
    AgentActionProto action_info = 2;
}
